### DISCUSSION TITLE
torch-directml has new release

### DISCUSSION THREAD

**Discussion by ChiaYen-Kan** [vladmandic/sdnext#3158] [General]
Hi
torch-directml is release new version at 2024-05-22
![new-torch-directml](https://github.com/vladmandic/automatic/assets/40080620/f2e61851-2ad5-4061-9c81-d82e91ff0b31)

i try to upgrade torch-directml by manual, but is can't work
has anybody is success upgrade to new torch-directml?

thank you

**Comment by vladmandic** [comment]
`webui.bat --use-directml --reinstall`
